src/API/OnboardingTasks.php

```diff
@@ -610,4 +641,230 @@ public function get_status() {
 
 		return rest_ensure_response( $status );
 	}
+
+	/**
+	 * Get the onboarding tasks.
+	 *
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function get_tasks() {
```

Comment:
This function will get cluttered over time, with all of the queries/logic regarding every task living here. Not a necessary optimization yet, but we may want to modularize the logic per-task at some point as the list grows more complex.

Reply:
Completely agree.  This along with most of the onboarding feature classes could benefit from being split up into smaller, task-related classes.

I think I may try and tackle this during next cooldown.